### PR TITLE
Add documentation for Jandy iAqualink component.

### PR DESCRIPTION
**Pull request by flz**
**Description:**

The iaqualink component adds support for Jandy pool control systems and exposes thermostats, lights, sensors and switches.

**Pull request in home-assistant (if applicable):** 
- home-assistant/home-assistant#26034 
- home-assistant/home-assistant#26484

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10165"></a>

### CODE REVIEW

**Comment by flz** [comment]
Pull request for the actual component will be submitted in a few minutes.

File: source/_components/iaqualink.markdown [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Jandy iAqualink"
+description: "Instructions on how to configure Jandy iAqualink integration."
+logo: iaqualink.png
+ha_category:
+  - Climate
+  - Light
+  - Sensor
+  - Switch
+ha_release: 0.97
```
Comment (klaasnicolaas) [inline source/_components/iaqualink.markdown]:
```suggestion
ha_release: 0.98
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/iaqualink.markdown [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Jandy iAqualink"
+description: "Instructions on how to configure Jandy iAqualink integration."
+logo: iaqualink.png
+ha_category:
+  - Climate
+  - Light
+  - Sensor
+  - Switch
+ha_release: 0.98
```
Comment (MartinHjelmare) [inline source/_components/iaqualink.markdown]:
```suggestion
ha_release: 0.99
```

File: source/_components/iaqualink.markdown [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Jandy iAqualink"
+description: "Instructions on how to configure Jandy iAqualink integration."
+logo: iaqualink.png
+ha_category:
+  - Climate
+  - Light
```
Comment (MartinHjelmare) [inline source/_components/iaqualink.markdown:7]: We only added the climate platform in this first PR.

File: source/_components/iaqualink.markdown [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+ha_release: 0.98
+ha_iot_class: Cloud Polling
+---
+
+[iAqualink](https://www.iaqualink.com/) by [Jandy](https://www.jandy.com/) allows you to control your pool anytime, anywhere.
+
+There is currently support for the following device types within Home Assistant:
+
+- Climate
+- Light
```
Comment (MartinHjelmare) [inline source/_components/iaqualink.markdown:17]: See above.

File: source/_components/iaqualink.markdown [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+```
+
+{% configuration %}
+username:
+  description: The username for accessing your iAqualink account
+  type: string
+  required: true
+  default: None
```
Comment (MartinHjelmare) [inline source/_components/iaqualink.markdown]: If there's no default, leave out this item.

File: source/_components/iaqualink.markdown [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+password:
+  description: The password for accessing your iAqualink account
+  type: string
+  required: true
+  default: None
```
Comment (MartinHjelmare) [inline source/_components/iaqualink.markdown]: See above.

File: source/_components/iaqualink.markdown [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+If you have problems with iAqualink or the integration you can add debug prints to the log.
+
+```yaml
+logger:
+  default: info
+  logs:
+    iaqualink: debug
+    homeassistant.components.iaqualink: debug
+    homeassistant.components.climate.unifi: debug
+    homeassistant.components.light.unifi: debug
```
Comment (MartinHjelmare) [inline source/_components/iaqualink.markdown]: Only the climate platform has been added.

**Review by MartinHjelmare — DISMISSED** [review]

**Comment by flz** [comment]
Re: climate-only: What's the grace period to get this documentation landed? I have all the other platforms ready to review (light platform PR has been submitted already) but they can't all be merged at the same time given that the patched need to be applied in order. I was hoping all platforms would make it to 0.99.

**Comment by MartinHjelmare** [comment]
I'll let the docs team respond.

**Comment by frenck** [comment]
⚠️ This cannot be merged since this PR has multiple parent PR's (and not all are merged).

@flz Please split the documentation PR's to match the parent PR's.

**Comment by flz** [comment]
> @flz Please split the documentation PR's to match the parent PR's.

I've removed mentions of sensor/switch platforms.

I've left the light platform mentions since the PR is ready for landing (home-assistant/home-assistant#26484).

**Comment by flz** [comment]
@frenck Should be good to go now!

**Review by frenck — APPROVED** [review]
Thanks, @flz! 👍 
Let's merge this in! 🎉

**Pull request merged**